nessie/sql_templates/create_advisor_schema.template.sql

```diff
@@ -174,10 +174,11 @@ AS (
         I.ADVISOR_TYPE_DESCR AS advisor_type,
         I.INSTRUCTOR_TYPE AS instructor_type_code,
         I.INSTRUCTOR_TYPE_DESCR AS instructor_type,
-        P.PERMISSION_LIST AS cs_permissions,
-        P.AUTH_ACTIONS AS cs_auth_actions
```

Comment:
Removed AUTH_ACTIONS because I didn't include that field in the file in S3 (since the value appears to be meaningless).